src/main/java/reposense/RepoSense.java

```diff
@@ -51,7 +50,7 @@ public static void main(String[] args) {
             RepoConfiguration.setFormatsToRepoConfigs(configs, cliArguments.getFormats());
             RepoConfiguration.setDatesToRepoConfigs(configs, cliArguments.getSinceDate(), cliArguments.getUntilDate());
             ReportGenerator.generateReposReport(configs, cliArguments.getOutputFilePath().toAbsolutePath().toString(),
-                    formatter.format(ZonedDateTime.now(ZoneId.of("UTC+8"))));
+                    formatter.format(ZonedDateTime.now(cliArguments.getZoneId())));
```

Comment:
Then you can simplify `formatter.format(ZonedDateTime.now(cliArguments.getZoneId())`
to `cliArguments.getZoneId()`.

However, I leave this as a **suggestion**. I'm okay as it is as this change may make testing harder.